docs/deployment/cumulus-distribution.md

```diff
@@ -0,0 +1,15 @@
+---
+id: cumulus_distribution
+title: Using the Cumulus Distribution API
+hide_title: false
+---
+
+The Cumulus Distribution API is a set of endpoints that can be used to enable AWS Cognito authentication when downloading data from S3.
+
+## Configuring a Cumulus Distribution deployment
+
+The Cumulus Distribution API is included in the main [Cumulus](https://github.com/nasa/cumulus/tree/master/tf-modules/cumulus_distribution) repo.
+
+To configure a deployment to use Cumulus Distribution, remove or comment the "Thin Egress App Settings" in [the Cumulus Template Deploy](https://github.com/nasa/cumulus-template-deploy/blob/master/cumulus-tf/main.tf) and enable the Cognito settings.
+
+No additional configuration is necessary to deploy the Cumulus Distribution API.
```

Comment:
They need to provide a bucket map file for cumulus_distribution as they do for TEA, and a bucket map file will be put to s3.
https://github.com/nasa/cumulus/tree/master/example/cumulus-tf/cumulus_distribution
In our example, we either take a bucket_map.yaml or bucket_map.yaml.tmpl
https://github.com/nasa/cumulus/blob/master/example/cumulus-tf/cumulus_distribution.tf#L3-L15

Also we need to call out cumulus_distribution module will generate a distribution bucket map cache file, and overwrites the bucket map cache file generated by distribution(TEA) module if any.  The bucket map cache files generated by cumulus_distribution and distribution module should be the same if they are using the same bucket map file.
cc @Jkovarik

Reply:
Sure, have a look at this:

https://github.com/nasa/cumulus/pull/2353/commits/d29db71cf327614278cce46dff742a56f1d4a701

Parts were taken from `thin-egress-app.md`. Is it missing anything?